index.js

```diff
@@ -1,356 +1,558 @@
+//   function returnSquare(number) {
+//      return number ** 2 ;
+//   }
 
-// const obj = {
-//   "Коля": 200,
-//   "Вася": 300,
-//   "Петя": 400,
-// };
+//   const square = returnSquare(11);
+//   console.log (square);
 
-// for (const key in obj) {
-//     console.log (`${key} — зарплата ${obj[key]} долларов.`)
-// }
+// // задание 4
 
-// // задание 5
+// function userAge() {
+    
+//     let age = prompt('Сколько вам лет');
+//     if (age < 0) {
+//         return (`Вы ввели неправильное значение`)
+//     } else if (age <= 12) {
+//         return (`Привет друг!`)
+//     } else {
+//         return (`Добро пожаловать`)
+//     }
+// }
+// alert (userAge())
 
-// let n = 1000;
-// let num = 0;
+// // задание 5 
 
-// while (n >= 50) { 
-//   n = n / 2;
-//   num++
+// function mult(c, f) {
+//     if (isNaN(c) || isNaN(f)) {
+//         return 'Одно или оба значения не являются числом'
+//     } else {
+//         return c * f 
+//     }
 // }
-// console.log(n);
-// console.log(num);
+// console.log (mult(12 , 5))
+// console.log (mult('n', 's'))
+
+// // задние 6
+
+// function cubeNumber() {
+//     const number = prompt('Введите число:', '');
+//     if (isNaN(number)) {
+//       return 'Переданный параметр не является числом';
+//     } else {
+//       const result = number ** 3;
+//       return `${number} в кубе равняется ${result}`;
+//     }
+//   }
+// console.log(cubeNumber());
+// // // console.log(cubeNumber(0)); // 0 в кубе будет 0
+// // // console.log(cubeNumber(5)); //5 кубе будет 125
+// // // console.log(cubeNumber(10)); // 10 в кубе будет 1000
+// // // console.log(cubeNumber('abc')); // не являктся числом
 
-// // задание 6
-// for (let friday = 3; friday <= 31; friday = friday + 7) {
-//   console.log(`Сегодня пятница, ${friday}-e число. Необходимо подготовить отчет.`);
-// }
+// // задание 7
 
-// HomeWork-JS-5
+// let circle1 = {
+//     radius: 5,
+//     getArea: function() {
+//       return Math.PI * this.radius ** 2;
+//     },
+//     getPerimeter: function() {
+//       return 2 * Math.PI * this.radius;
+//     }
+//   };
+  
+//   let circle2 = {
+//     radius: 7,
+//     getArea: function() {
+//       return Math.PI * this.radius ** 2;
+//     },
+//     getPerimeter: function() {
+//       return 2 * Math.PI * this.radius;
+//     }
+//   };
 
-// задание 1
+// console.log(circle1.getArea()); 
+// console.log(circle1.getPerimeter()); 
 
-function min (a, b) {
-    
-  return a < b ? a : b;
-}
-console.log (min(8,4))
-console.log (min(6,6))
+// console.log(circle2.getArea()); 
+// console.log(circle2.getPerimeter()); 
 
+// Home-Work 2.6
 
-// задание 2
+//  задание 1
 
-function num(number) {
-    return number % 2 == 0 ? 'четное' : 'нечетное'
-    
+
+const numbs = [1, 5, 4, 10, 0, 3];
+
+for (let i = 0; i < numbs.length; i++) {
+	if (numbs[i] == 10) break;
+	console.log(numbs[i]);
+}
+for (let a = 0; a < numbs.length; a++) {
+	if (numbs[a] < 1) break;
+	console.log(numbs[a]);
 }
```

Comment:
for (let i = 0; i < numbs.length; i++) {
	console.log(numbs[i]);
	if (numbs[i] == 10) break;
}